apps/todo-list-client/src/app/generic-button/generic-button.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-generic-button',
+  templateUrl: './generic-button.component.html',
+})
+export abstract class InputButtonUnitComponent {
+  abstract title: string;
+
+  abstract changeAction(): void;
```

Comment:
Maybe a better name is onClick for this method, please consider that

Reply:
I'm done with this, thank you